build/common.project.props

```diff
@@ -23,6 +23,7 @@
     <SolutionPackagesFolder>$(RepositoryRootDirectory)packages\</SolutionPackagesFolder>
     <XunitConsoleExePath>$(SolutionPackagesFolder)xunit.runner.console.2.2.0\tools\xunit.console.x86.exe</XunitConsoleExePath>
     <ILMergeExePath>$(SolutionPackagesFolder)ILMerge.2.14.1208\tools\ILMerge.exe</ILMergeExePath>
+    <ILRepackExePath>$(SolutionPackagesFolder)ILRepack.2.0.13\tools\ILRepack.exe</ILRepackExePath>
```

Comment:
Package contains the props file defining the tool location. Also would it work if installed directly in target project?